### ISSUE TITLE
Have the documentation system support LaTeX formulas

### ISSUE THREAD

**Issue by vkk800** [keras-team/keras#10060]
As far as I can tell, it's not possible to write formulas in LaTeX in the documentation. In my opinion, it would be useful to add formulas to documentations of e.g. optimizers and activations. At the moment, some parts of the documentation write formulas as code, which is a bit ugly and would be even uglier for more complicated formulas.

There exists a MathJax extension to MkDocs that could be used, see: https://stackoverflow.com/questions/27882261/mkdocs-and-mathjax .

EDIT: Apparently the same issue has already been posted here, but it was closed due to inactivity, see https://github.com/keras-team/keras/issues/2911

**Comment by vkk800** [comment]
@Dref360 I was planning I might look into it on the next non-sunny day around here. I'm not sure, though, if it needs something installed on the server that is hosting the docs. If that is the case, I can't do it.

**Comment by vkk800** [comment]
@Dref360 I added the support to latex formulas on my local repository, it was rather easy. The problem is that I had to install python-markdown-math extension first. It has to be located in whatever path mkdocs looks for python libraries so simply copying the .py file to Keras docs/ folder did not work. Installing the extension was as easy as just doing `pip3 install python-markdown-math`.

Unless there are suggestions on how to go around this, I think whoever maintains the Keras.io server needs to install this extension and put it in the correct path.
